app/src/main/java/org/oppia/android/app/shim/ViewBindingShim.kt

```diff
@@ -46,6 +48,19 @@ interface ViewBindingShim {
     profileInputView: View
   ): TextView
 
+  /** Handles binding inflation for [PromotedStoryListView] */
+  fun inflatePromotedStoryCardBinding(
+    inflater: LayoutInflater,
+    parent: ViewGroup,
+    attachToParent: Boolean
+  ): View
+
+  /** Handles binding inflation for [PromotedStoryListView] and returns the view model.*/
```

Comment:
Add space just before `.`

`/** Handles binding inflation for [PromotedStoryListView] and returns the view model. */`

Reply:
Done